docs/platform-integration/includes/ios-PresentationSourceBounds.md

```diff
@@ -3,16 +3,19 @@ ms.topic: include
 ms.date: 02/02/2023
 ---
 
-When requesting a share or opening launcher on iPadOS, you can present it in a popover. This specifies where the popover will appear and point an arrow directly to. This location is often the control that launched the action. You can specify the location using the <xref:Microsoft.Maui.ApplicationModel.OpenFileRequest.PresentationSourceBounds> property:
+> [!IMPORTANT]
+> This section only applies to iOS.
+
+When requesting a share or opening launcher on iOS, you can present it in a popover. This specifies where the popover will appear and point an arrow directly to. This location is often the control that launched the action. You can specify the location using the <xref:Microsoft.Maui.ApplicationModel.OpenFileRequest.PresentationSourceBounds> property:
```

Comment:
Afraid it needs to be altered back to iPadOS (rather than iOS). The OS's are called iPhoneOS and iPadOS. When you request a share/launcher on iPhoneOS, you have no control over where it appears. It appears on a sheet at the bottom of the screen. However, when you request a share/launcher on iPadOS, it appears as a popup and you can control where on screen it appears. So that's what that include file is about.

Reply:
```suggestion
When requesting a share or opening launcher on iPadOS, you can present it in a popover. This specifies where the popover will appear and point an arrow directly to. This location is often the control that launched the action. You can specify the location using the <xref:Microsoft.Maui.ApplicationModel.OpenFileRequest.PresentationSourceBounds> property:
```